helix-vte/src/pty/command.rs

```diff
@@ -0,0 +1,176 @@
+pub struct Command {
+    inner: tokio::process::Command,
+    stdin: bool,
+    stdout: bool,
+    stderr: bool,
+    pre_exec_set: bool,
+    pre_exec: Option<Box<dyn FnMut() -> std::io::Result<()> + Send + Sync + 'static>>,
+}
+
+impl Command {
+    /// See [`tokio::process::Command::new`]
+    pub fn new<S: AsRef<std::ffi::OsStr>>(program: S) -> Self {
+        Self {
+            inner: tokio::process::Command::new(program),
+            stdin: false,
+            stdout: false,
+            stderr: false,
+            pre_exec_set: false,
+            pre_exec: None,
+        }
+    }
```

Comment:
Why the wrapper type? `Command::new` is already a builder function

Reply:
this is a part of `pty_process` crate I am intended to ether move it into own crate or wait while maintainer to fix `pty_process`